components/integrations/abstract_incoming_webhook.jsx

```diff
@@ -73,6 +73,7 @@ export default class AbstractIncomingWebhook extends React.PureComponent {
             description: hook.description || '',
             channelId: hook.channel_id || '',
             channelLocked: hook.channel_locked || false,
+            enabled: hook.enabled ?? true,
```

Comment:
this will pass falsey values such as empty strings and so on to the property as well ... it would be better to use the `||` here

Reply:
enabled is a bool so it shouldn't ever be an empty string, if we use || then it will always be true and the checkbox when editing the webhook will always be checked, even if the webhook was disabled.

Reply:
Yes, it shouldn't ... it's still safer to do it like that, so please change it

Reply:
Good catch. I just forgot to mention to turn around the condition. 

You basically just use 'enabled || false' ... that way the behavior is correct ... thinking about it: 'Boolean(var)' would be totally sufficient. 😂

Reply:
@mickmister ^^

Reply:
@michelengelen That still wouldn't result in the correct behavior though. If `hook.enabled` is `undefined` we want `state.enabled` to be `true`, but `enabled || false` and `Boolean(var)` would both result in `state.enabled` being `false`